subsys/bluetooth/audio/shell/audio.h

```diff
@@ -284,7 +284,7 @@ static void print_ltv_array(const struct shell *sh, size_t indent, const uint8_t
 		.indent = indent,
 	};
 
-	bt_audio_data_parse(ltv_data, ltv_data_len, print_ltv_elem, &ltv_info);
+	(void)bt_audio_data_parse(ltv_data, ltv_data_len, print_ltv_elem, &ltv_info);
```

Comment:
`bt_audio_data_parse` may still return an error even if `print_ltv_elem` never does (in the case that we receive invalid LTV encoded data from a peer device. 

I think a better solution would actually be to output that to the shell as an error

Reply:
Done.